apps/challenges/challenge_notification_util.py

```diff
@@ -0,0 +1,36 @@
+import logging
+
+from django.conf import settings
+
+from base.utils import send_email
+from challenges.aws_utils import start_workers
+
+logger = logging.getLogger(__name__)
+
+
+def challenge_start_notifier(sender, instance, field_name, **kwargs):
+    prev = getattr(instance, "_original_{}".format(field_name))
+    curr = getattr(instance, "{}".format(field_name))
+    if curr and not prev:  # Checking if the challenge has been approved by admin since last time.
+        challenge = instance
+
+        # Start the challenge worker.
+        response = start_workers([challenge])
```

Comment:
@KhalidRmb This starting of challenge worker should only happen when the challenge is not docker based.

We should send the email to challenge host in both the cases.

Reply:
Done.